os/drivers/lcd/lcd_dev.c

```diff
@@ -93,9 +96,17 @@ static int lcddev_close(FAR struct file *filep);
 static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg);
 static ssize_t lcddev_read(FAR struct file *filep, FAR char *buffer, size_t buflen);
 static ssize_t lcddev_write(FAR struct file *filep, FAR const char *buffer, size_t buflen);
+
 /****************************************************************************
  * Private Data
  ****************************************************************************/
+static uint8_t *lcd_kbuffer;
```

Comment:
These are temporal implementation with global variables because of tight schedule.
So I think it is not good. Please implement flushing behavior in a proper method without using global variables if possible.